semgrep-core/src/parsing/ast/Loc.mli

```diff
@@ -0,0 +1,79 @@
+(*
+   Representation of the location of a piece of code by a pair of tokens.
+
+   This is used so far in intermediate, language-specific ASTs.
```

Comment:
Maybe explain why we need that? Why only Bash/Dockerfile are using that?
Which advantages does that bring over using Parse_info.t?